src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesWriter.groovy

```diff
@@ -40,9 +41,25 @@ class JcrNodesWriter implements ItemWriter<NodeProtos.Node>, ItemWriteListener {
         ServletOutputStream servletOutputStream = theServletOutputStream()
         if (servletOutputStream == null) throw new IllegalStateException("servletOutputStream must be set.")
 
-        nodeProtos.each { NodeProtos.Node node ->
-            log.debug "Sending NodeProto : ${node.name}"
-            node.writeDelimitedTo(servletOutputStream)
+        try {
+            //log.info "\n\n### NodeProtos ###\n\n${StringUtils.join(nodeProtos.toString(), "\n")}"
```

Comment:
We can get rid of all the commented out code (here and elsewhere in the PR). Thanks!

Reply:
No problem will do